cypress/integration/required-metadata.spec.js

```diff
@@ -104,6 +104,15 @@ describe('Required Metadata Page', () => {
         cy.contains('my default title');
       });
   });
+
+  it('Organization is pre-selected in the dropdown if the form is loaded from organization page', () => {
+    cy.login();
+    cy.visit('/organization/test-123');
```

Comment:
@kmanaseryan I would expect `?group=...` parameter here

Reply:
@anuveyatsu it visits organization page then clicks on the button "Add dataset" below

Reply:
oh, you are right @kmanaseryan